projects/RabbitMQ.Client/client/impl/ChannelBase.cs

```diff
@@ -571,6 +571,26 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
+        public async ValueTask DisposeAsync()
+        {
+            await DisposeAsyncCore()
+                .ConfigureAwait(false);
+
+            Dispose(false);
+        }
+
+        protected virtual async ValueTask DisposeAsyncCore()
```

Comment:
It seems we are currently not really overriding dispose in the derived type. Should we simplify this or is it worth keeping things extensible? Or maybe it is a topic for another PR because technically the whole channel base could be inlined into channel

Reply:
How about simplifying in another PR?